recipes/ujson/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ujson" %}
+{% set version = "1.35" %}
+{% set sha256 = "f66073e5506e91d204ab0c614a148d5aa938bdbf104751be66f8ad7a222f5f86" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+
+  run:
+    - python
+
+test:
+  imports:
+    - ujson
+
+about:
+  home: https://github.com/esnme/ultrajson
+  summary: "Ultra fast JSON decoder and encoder written in C with Python bindings"
+  license: BSD
+  # license_file: LICENSE.txt
```

Comment:
Here too, please [package the license manually](http://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually) and file an issue or send a PR upstream to add it to `MANIFEST.in` ([example](https://github.com/bokeh/colorcet/pull/6)).